gamefixes-gog/umu-1580232252.py

```diff
@@ -0,0 +1,8 @@
+""" Resident Evil (1997)
+"""
+#pylint: disable=C0103
+
+from protonfixes import util
+
+def main():
+    util.winedll_override('ddraw', 'n,b')
```

Comment:
```suggestion
    util.winedll_override('ddraw', 'n,b')
    util.winedll_override('dinput', 'n,b')
```
For the controller fixes.